packages/dev/core/src/Rendering/geometryBufferRenderer.ts

```diff
@@ -610,6 +684,15 @@ export class GeometryBufferRenderer {
                 defines.push("#define NORMAL_INDEX " + this._normalIndex);
                 defines.push("#define PREPASS_NORMAL");
             }
+        } else {
+            if (this._enableDepth) {
+                defines.push("#define DEPTH");
+                defines.push("#define DEPTH_INDEX " + this._depthIndex);
+            }
+            if (this._enableNormal) {
+                defines.push("#define NORMAL");
+                defines.push("#define NORMAL_INDEX " + this._normalIndex);
+            }

```

Comment:
We can simplify things quite a bit now that depth and normal are not special anymore. You can replace the whole `if (this._linkedWithPrePass) { ... } else { ... } ` block with:

```typescript
            if (this._enableDepth) {
                defines.push("#define DEPTH");
                defines.push("#define DEPTH_INDEX " + this._depthIndex);
            }
            if (this._enableNormal) {
                defines.push("#define NORMAL");
                defines.push("#define NORMAL_INDEX " + this._normalIndex);
            }
```

Reply:
Good call. Done.